src/libraries/System.Net.Security/src/System/Net/Security/SslStreamCertificateContext.Linux.cs

```diff
@@ -247,7 +247,6 @@ private SslStreamCertificateContext(X509Certificate2 target, ReadOnlyCollection<
                         _ocspResponse = ret;
                         _ocspExpiration = expiration;
                         _nextDownload = nextCheckA < nextCheckB ? nextCheckA : nextCheckB;
-                        _pendingDownload = null;
                         break;
                     }
                 }
```

Comment:
I'm wondering if we should add `else` with something like `_nextDownload = 5m` (or maybe hour..)
That would prevent immediate retry and possibly hammering server.

and since we touching this, perhaps also add tracing for cases when download was not successful.

Reply:
> I'm wondering if we should add else with something like _nextDownload = 5m (or maybe hour..)
That would prevent immediate retry and possibly hammering server.

It already serializes the requests (won't do more than one in parallel), and (as @rzikm observed when opening the PR) it failing here just mean clients are going to turn around and ask the OCSP server directly... so not doing any backoff is probably the best state.

So I advise against having a backoff, or only having it at like 5 seconds, not 5 minutes.

Reply:
we should be checking new OCSP one every day or 5 minutes before expiration, right? 5s seem aggressive to me. But it is probably still better than staying on broken unrecoverable state.

Reply:
I added 5s back-off after unsuccessful query.